desktop_projects/desktop_sim/tests/test_expressions.cpp

```diff
@@ -9,18 +9,28 @@
 #include "expression_store.h"
 #include "hardware_manager.h"
 
-TEST(ExpressionStorage, expressionParsing) {
+class ExpressionStorageTests : public testing::Test {
+ protected:
+  void SetUp() override {
+    hm_hardwareInit();
+    cli_init();
+    cli_setDefaultConfig();
+    expressionStore.init();
+    expressionStore.removeExpressionsForTrigger(0);
+    expressionStore.removeExpressionsForTrigger(1);
+    expressionStore.removeExpressionsForTrigger(2);
```

Comment:
why are these here? (just document)

Reply:
They were in there from when there would be default configurations set by the CLI if it didn't detect any written configuration but that doesn't happen anymore so will remove.